src/object.c

```diff
@@ -912,6 +912,21 @@ int getPositiveLongFromObjectOrReply(client *c, robj *o, long *target, const cha
     }
 }
 
+/* Sends either a reply or an error reply by checking the first char.
+ * If the first char is '-' the reply is considered an error.
+ * In any case the given reply is sent, if the reply is also recognize
+ * as an error we also perform some post reply operations such as
+ * logging and stats update. */
+void senReplyOrError(client *c, robj *reply) {
```

Comment:
you were missing a `d` in `send`.
but i wanna suggest exchanging `send` with `add` since that's the output buffer terminology we use.
and also add an `0bject` suffix since maybe we'll have a similar method one day that takes a `char*` so for the most part, the current convention (at least specifically for errors) dictates that the one that takes `robj` is suffixed with `Object`.
also, ironically, that's what it does (either `addReply` or `addReplyErrorObject`).

@madolson any better suggestion?
do you think such a method belongs in networking.c (near `addReplyErrorObject`) or in object.c (near getPositiveLongFromObjectOrReply`), maybe both belong in networking.c, but that may be a topic for another day.
```suggestion
void addReplyOrErrorObject(client *c, robj *reply) {
```

Reply:
Fixed

Reply:
I think this function should be in networking.c, it is one of the foundational functions for adding replies onto clients.